### PR TITLE
LPS-33814 My Activity TestCases

### PR DESCRIPTION
**Pull request by AkramYoussoufi**

### CODE REVIEW

**Comment by AkramYoussoufi** [comment]
ci:test:sf

**Comment by pedroamarall** [comment]
ci:test:sf

**Comment by pedroamarall** [comment]
> ### ❌ ci:test:sf - 0 out of 1 jobs passed in 4 minutes
> Click here for more details.
> #### Base Branch:
> Branch Name: [master](https://github.com/liferay/liferay-portal/tree/master)Branch GIT ID: [2f31084975b8217be699b430883a9f0a97c1eefc](https://github.com/liferay/liferay-portal/commit/2f31084975b8217be699b430883a9f0a97c1eefc)
> 
> #### Sender Branch:
> Branch Name: [LPS-33814](https://github.com/AkramYoussoufi/liferay-portal/tree/LPS-33814)Branch GIT ID: [d4d7b6d37dedbf083f678e13e9fa2575aa669da1](https://github.com/AkramYoussoufi/liferay-portal/commit/d4d7b6d37dedbf083f678e13e9fa2575aa669da1)
> 
> 0 out of 1jobs PASSED
> #### 1 Failed Jobs:
> * [test-portal-source-format](https://test-1-15.liferay.com/job/test-portal-source-format/4842/)
> 
> ##### For more details click [here](https://test-1-15.liferay.com/userContent/jobs/test-portal-source-format/builds/4842/jenkins-report.html).
>      [java] java.lang.Exception: Found formatting issues:
>      [java] At least one commit message is missing a reference to a required JIRA project: BLADE, CLDSVCS, COMMERCE, FACES, IDE, ISSITES, LCD, LHC, LOOP, LPS, LRAC, LRCI, LRDOCS, LRIS, LRQA, OAUTH2, POSHI, RELEASE, SYNC, TR. Please verify that the JIRA project keys are specifiedin ci.properties in the liferay-portal repository.
>      [java]   at com.liferay.source.formatter.util.JIRAUtil.validateJIRAProjectNames(JIRAUtil.java:44)
>      [java]   at com.liferay.source.formatter.SourceFormatter._validateCommitMessages(SourceFormatter.java:1298)
>      [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:326)
>      [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:290)
> [stopwatch] [run.batch.test.action: 1:24.313 sec]
>      [echo] The following error occurred while executing this line:
>      [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:504: The following error occurred while executing this line:
>      [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:648: Java returned: 1
>       [get] Getting: http://test-1-15/job/test-portal-source-format/4842//consoleText
>       [get] To: /opt/dev/projects/github/liferay-portal/20230922085041907.txt
>    [delete] Deleting: /opt/dev/projects/github/liferay-portal/20230922085041907.txt
>   [typedef] Could not load definitions from resource org/apache/maven/artifact/ant/antlib.xml. It could not be found.
>   [taskdef] Could not load definitions from resource org/jacoco/ant/antlib.xml. It could not be found.
>    [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1161358896.properties
> merge-test-results:
> [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
> [beanshell] Truncating errors in /opt/dev/projects/github/liferay-portal/portal-impl/test-results/TEST-JenkinsLogAssertTest.xml
> [junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null1598265846
> [junitreport] Loading stylesheet jar:file:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
> [junitreport] Transform time: 215ms
> [junitreport] Deleting: /tmp/null1598265846
> [echo] A report with all the test results can be found at test-results/html/index.html.
> [mkdir] Created dir: /opt/dev/projects/github/liferay-jenkins-ee/test-results

Hey @AkramYoussoufi Can you fix this problem about the name of the commit, I will start to review your codes when you fix this problem about source format, if you need some help, let me know please

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        task("Given The DM Portlet is added to the page"){
+           DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+        }
+        task("When The user create a Documents and Media (Upload the file)"){
+           JSONDocument.addFileWithUploadedFile(
+            dmDocumentTitle = ${dmDocumentTitle},
+            groupName = "Guest",
+            mimeType = "image/png",
+            sourceFileName = "really-youness.png"
+            );
```
Comment (marcosvictor22) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase:33]: Need space between lines

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    @priority = "5"
+    test CanEditDM {
+        task("the DM Portlet is added to the page and  create Document and Media (Upload the file)"){
+            JSONDocument.addFileWithUploadedFile(
+            dmDocumentTitle = ${dmDocumentTitle},
+            groupName = "Guest",
+            mimeType = "image/png",
+            sourceFileName = "really-youness.png"
+            );
+           DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
```
Comment (marcosvictor22) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase:54]: The same

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                AssertElementPresent(
+                key_dmDocumentTitle="Should Not Be Deleted",
+                locator1="DocumentsAndMedia#DESCRIPTIVE_LIST_DOCUMENT_TITLE"
+                );
+        }
+    }
+
+    @description = "Access Portal Global Menu and validate CONFIGURATION string at Control Panel!"
+    @priority = "5"
+    test CanDownloadDM {
```
Comment (marcosvictor22) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase:111]: Need to use BDD method

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        task("File is edited"){
+            AssertElementPresent(
+                key_dmDocumentTitle="Edited Title",
+                locator1="DocumentsAndMedia#DESCRIPTIVE_LIST_DOCUMENT_TITLE"
+            );
+        }
+    }
+
+    @description = "Delete File From DM"
+    @priority = "5"
```
Comment (marcosvictor22) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase:73]: Need to use BDD method

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+    tearDown {
+        User.logoutAndLoginPG(
+            userLoginEmailAddress = "test@liferay.com",
+            userLoginFullName = "Test Test");
+    }
+
+    @description = "Upload File to DM"
+    @priority = "5"
+    test CanCreateDM {
```
Comment (marcosvictor22) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDocumentAndMediaTest.testcase:23]: Tip: run the source format before sending a push again to pass the so ci:test:sf compilation in pedro's repository

**Pull request closed without merging**